Update Unit Tests (#78)
Orbital-Knewbie/Knewbie: docs/DeveloperGuide.md

More information on the parameters can be found on [CatSim's documentation](https://douglasrizzo.com.br/catsim/introduction.html). 

> > **_DISCLAIMER:_**  The proficiency and parameters used in Knewbie's current implementation have no research behind them. Since the development team has limited resources, the CatSim provided function `catsim.cat.generate_item_bank` was used to create simulated parameters for the questions, and `catsim.initialization.RandomInitializer().initialize()` was used to generate simulated proficiencies for each user. In the future post-Orbital, it may be possible to obtain a much larger dataset with more accurate parameters.

Reviewer: ```suggestion
> **_DISCLAIMER:_**  The proficiency and parameters used in Knewbie's current implementation have no research behind them. Due to the limited resources of the development team, the function provided by CatSim, `catsim.cat.generate_item_bank`, was used to create simulated parameters for the questions, and `catsim.initialization.RandomInitializer().initialize()` was used to generate simulated proficiencies for each user. In the future, post-Orbital, it may be possible to obtain a much larger dataset with more accurate and researched parameters.
```
